english/watched/watched.md

```diff
@@ -1,4 +1,3 @@
 You gaze into the abyss, and the abyss gazes into you.
 
-Perhaps you'd like to lighten up and [bake some cookies.](../bake-cookies/cookies.md)
-
+Perhaps you'd like to lighten up and [bake some cookies.](../bake-cookies/cookies.md) Or, you may decide it's easier just to go to the bakery and [buy them.](buy-cookies/buy-cookies.md)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.